fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeCheckerV2.java

```diff
@@ -195,5 +208,92 @@ public Void visitAlterResourceStatement(AlterResourceStmt statement, ConnectCont
             }
             return null;
         }
+
+        // ---------------------------------------- Show Node Info Statement---------------------------------------------
+        @Override
+        public Void visitShowBackendsStatement(ShowBackendsStmt statement, ConnectContext context) {
+            return checkShowNodePrivilege(context);
```

Comment:
Should check for either `node` or `operate`. Same with `show frontends`, `show brokers`.

Reply:
done